src/s3fetchers.ts

```diff
@@ -21,12 +21,13 @@ import { normalizeBlockHeight, parseBody } from "./utils";
 export async function listBlocks(
   client: S3Client,
   bucketName: string,
-  startAfter: BlockHeight
+  startAfter: BlockHeight,
+  limit = 10
```

Comment:
Let's default to 200 since it will make less requests, and boosts the throughput quite a bit.

@khorolets Let's make this parameter configurable on Rust side as well and update the default to 200 (somehow we used 100 there, but choose to use 10 in JS version)

Reply:
looks like 200 might be a bit suboptimal when dealing with more meaty blocks, might need experiments on user's side to tune – e.g. at block  #46661963 it seems that 100 is working better

maybe just needs another change to avoid blocking for all of them to be loaded